apps/src/templates/instructions/ImmersiveReaderButton.jsx

```diff
@@ -32,6 +32,7 @@ class ImmersiveReaderButton extends Component {
 
     return (
       <div
+        tabIndex="0"
```

Comment:
should we add the `role="button"` attribute as well to improve accessibility?

if we do, there's an interesting note from MDN about this in ["ARIA: button role"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role):

> Note: If using role="button" instead of the semantic <button> or <input type="button"> elements, you will need to make the element focusable and define event handlers for [click](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event) and [keydown](https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event) events. This includes handling the Enter and Space keypresses in order to process all forms of user input.

Reply:
good call out! I'm adding the role and adding a keydown event